recipes/rdma-core/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "rdma-core" %}
+{% set version = "28.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/linux-rdma/rdma-core/archive/v{{ version }}.tar.gz
+  sha256: 899761740efc45642ab3bc78e48b73374ce8cd17fcc9155ca44d358a94684b29
+  patches:
+    - 0001-Remove-assertions-on-longer-than-sockaddr_un.sun_pat.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ cdt("libnl3-devel") }}
+    - {{ cdt("systemd-devel") }}
+    - cmake
+    - make
```

Comment:
If we're using Ninja is there a reason we need `make` as well?

Reply:
No reason, that was an oversight, thanks for catching. As of https://github.com/conda-forge/staged-recipes/pull/22869/commits/a9125a8ff5abf9f1b87aab9dd3da776c95c80001 that is now removed.